src/common/form/example/index.js

```diff
@@ -0,0 +1,242 @@
+
+const ListLine = React.createClass({
+    mixins: [FocusComponents.list.selection.line.mixin],
+    definitionPath: "commande",
+    renderLineContent(data){
+        const firstName = this.displayFor("name", {});
+        const lastName = this.displayFor("number", {});
+        return <div>{firstName} {lastName}</div>;
+    }
+});
+
+const contactStore = new Focus.store.CoreStore({
+    definition: {
+        'contact': 'contact',
+        'commandes': 'commande'
+    }
+});
+
+const jsonContact= {
+    firstName: "Zeus",
+    lastName: "God",
+    isCool: true,
+    birthDate: new Date().toISOString(),
+    commandes: [{
+        name: "commande1",
+        number: "1"
+    }, {
+        name: "commande2",
+        number: "2"
+    }, {
+        name: "commande3",
+        number: "3"
+    }, {
+        name: "commande4",
+        number: "4"
+    }, {
+        name: "commande5",
+        number: "5"
+    }, {
+        name: "commande6",
+        number: "6"
+    }]
+};
+
+const action = {
+    load: actionBuilder({
+        status: 'loaded',
+        node: 'contact',
+        service(){
+            return new Promise(function(s,e){
+                _.delay(function(){
+                    s(jsonContact);
+                }, 5000);
+            })//Promise.resolve(jsonContact);
+        }
+    }),
+    save:actionBuilder({
+        status: 'saved',
+        preStatus: 'saving',
+        node: 'contact',
+        service(data){
+            console.log('save', data);
+            return Promise.resolve(data);
+        }
+    })
+};
+
+const FormExample = React.createClass({
+    displayName: "FormExample",
+    mixins: [formMixin],
+    stores: [{
+        store: contactStore,
+        properties: ["contact", "commandes"],
+    }],
+    definitionPath: "contact",
+    action: action,
+    referenceNames: ['papas', 'monkeys'],
+
+    /**
+     * Render content form.
+     * @return {ReactDOMNode} node REACT
+     */
+    renderContent() {
+        return (
+            <Block title="Fiche de l'utilisateur" actions={this._renderActions}>
+                {this.fieldFor("firstName")}
+                {this.fieldFor("lastName")}
+                {
+                    this.textFor("birthDate", {
+                        formatter: function (date) {
+                            return "formatted date" + date
+                        }
+                    })
+                }
+                {this.fieldFor('papaCode', {listName: 'papas'})}
+                {this.fieldFor('monkeyCode', {listName: 'monkeys', valueKey: 'myCustomCode', labelKey: 'myCustomLabel' })}
+                {this.fieldFor("bio")}
+                {this.fieldFor("isCool")}
+                {this.fieldFor("isNice")}
+                {this.fieldFor("birthDate")}
+                {this.listFor("commandes", {lineComponent: ListLine})}
+            </Block>
+        );
+    }
+});
+
+return (
+    <div>
+        <MessageCenter />
```

Comment:
maybe add a comment on the message center (i know it wasn't there in the initial example)